t/http_req_header.t

```diff
@@ -277,7 +277,26 @@ get request path: /t,
 
 
 
-=== TEST 15: get method
+=== TEST 15: get path (args)
+--- config
+location /t {
+    content_by_lua_block {
+        local wasm = require("resty.proxy-wasm")
+        local plugin = assert(wasm.load("plugin", "t/testdata/http_header/main.go.wasm"))
+        local ctx = assert(wasm.on_configure(plugin, 'req_path_get'))
+        assert(wasm.on_http_request_headers(ctx))
+    }
+}
+--- request
+GET /t?q=hello&a=world
+--- grep_error_log eval
+qr/get request path: \S+/
+--- grep_error_log_out
+get request path: /t?q=hello&a=world,
```

Comment:
is it wrong? I think the `path` should be `/t`

Reply:
:path gets url+args, which can be observed and verified through the browser devtools.

![image](https://user-images.githubusercontent.com/8529452/145224436-aca1dead-2c4a-4587-ad8a-542570769fe0.png)